packages/terminal/src/browser/base/terminal-widget.ts

```diff
@@ -38,6 +38,16 @@ export abstract class TerminalWidget extends BaseWidget {
 
     abstract onDidOpen: Event<void>;
 
+    abstract scrollLineUp(): void;
```

Comment:
A breaking change note is required (`changelog`) for these new `abstract` methods.
Anyone currently implementing `TerminalWidget` will be broken by the following changes.

Reply:
nobody should implement `TerminalWidget`, we should rather get rid of boilerplate interface later: https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#classes-over-interfaces